packages/flutter_tools/pubspec.yaml

```diff
@@ -11,7 +11,7 @@ dependencies:
   args: 2.3.0
   browser_launcher: 1.1.0
   dds: 2.2.0
-  dwds: 12.1.0
+  dwds: 13.0.0
```

Comment:
`dwds` major bump 
https://pub.dev/packages/dwds/changelog#1300

Reply:
thanks Jen, this what I was trying to do now:) That might require changes in devfs_web.dart